contracts/mockup/LockedSOVFailedMockup.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.5.17;
```

Comment:
I think we don't need this file. When we merge, this would create an exact replica of the file 'lockedSOVFailedMockup' which is not required.